stack.yaml

```diff
@@ -15,6 +16,10 @@ flags: {}
 # Extra package databases containing global packages
 extra-package-dbs: []
 
+allow-newer: true
+allow-newer-deps:
+  - scotty
```

Comment:
Reasoning:
scotty-cookie doesn't work with scotty >  0.12, so added upper bound.
But this older scotty has too-restrictive upper bounds on baseline dependencies like bytestring, etc.
Thus adding this allow-newer, but only limiting it to scotty's dependencies.